MoreLinq/Sequence.cs

```diff
@@ -17,31 +17,31 @@
 
 namespace MoreLinq
 {
-    using System;
     using System.Collections.Generic;
-    using System.Linq;
 
     static partial class MoreEnumerable
     {
         /// <summary>
-        /// Generates a sequence of integral numbers within the specified range.
+        /// Generates a sequence of integral numbers within the (inclusive) specified range.
+        /// If sequence is ascending the step is +1, otherwise -1.
         /// </summary>
         /// <param name="start">The value of the first integer in the sequence.</param>
         /// <param name="stop">The value of the last integer in the sequence.</param>
         /// <returns>An <see cref="IEnumerable{Int32}"/> that contains a range of sequential integral numbers.</returns>
         /// <example>
         /// <code>
-        /// string result = Sequence(6, 0);
+        /// var result = MoreEnumerable.Sequence(6, 0);
         /// </code>
         /// The <c>result</c> variable will contain <c>{ 6, 5, 4, 3, 2, 1, 0 }</c>.
         /// </example>
```

Comment:
Missing a [`<remarks>`](https://msdn.microsoft.com/en-us/library/3zw4z1ys.aspx) section about deferred execution semantics and streaming. [See `Batch`, for example](https://github.com/morelinq/MoreLINQ/blob/cc188dd124f947dd16d4e64ad7582d6bc8cfabf4/MoreLinq/Batch.cs#L52-L54).